pkg/storage/sds/edvformatprovider/edvdocumentprocessor/edvdocumentprocessor.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package edvdocumentprocessor
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/storage/sds"
+)
+
+// EDVDocumentProcessor represents a type that can encrypt and decrypt between
+// Structured Documents and Encrypted Documents.
+type EDVDocumentProcessor interface {
```

Comment:
is there going to be a default implementation of this interface?

Reply:
Yes - the default implementation will use the Aries JOSE Encrypter and Decrypter.